pkg/storage/metamorphic/generator.go

```diff
@@ -205,6 +206,10 @@ func (m *metaTestRunner) init() {
 	m.openBatches = make(map[readWriterID]storage.ReadWriter)
 	m.openTxns = make(map[txnID]*roachpb.Transaction)
 	m.openSavepoints = make(map[txnID][]enginepb.TxnSeq)
+	// TODO(travers): Add metamorphic test support for different versions, which
+	// will give us better coverage across multiple format major versions and
+	// table versions.
+	m.st = cluster.MakeTestingClusterSettings()
```

Comment:
I get really nervous when I see a `MakeTestingClusterSettings` in a non-_test.go file. I gather from looking around a little more that this is in fact a test file so I guess I shouldn't me. Part of me still wishes this was a parameter used to create the metaTestRunner so the make call was in a main or _test package all the same.

Reply:
Fair point! It was easy to move it to a `_test` file.

I'm thinking that addressing the TODO can probably remove the need for `MakeTestingClusterSettings` entirely, and instead set up a cluster version in a "more real" way.